### PR TITLE
Add UI to remove expenditure draft

### LINKED ISSUE #3544: Expenditures: As a user I can delete the expenditure
[Provided by Michał via Contractors project management platform]

## Description
User should be able to delete the draft.

## Acceptance Criteria

- an option to delete the draft
- user should accept deleting in the modal
- deleting expenditure is blocked after "locking values", instead they need to cancel the expenditure

## Design

- [Figma Link - Delete Draft](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=2875%3A335017)

Cancel by motion@2x12

### PR DESCRIPTION
**Pull request by agnieszkajarosikloj**
## Description

UI part of deleting expenditure draft. 
Screenshot 2022-06-29 at 22 37 56

[Figma link:](https://www.figma.com/file/dCtvv76S8mk5HNApjwmBto/Expenditures?node-id=2875%3A335017)

**New stuff** ✨

* DeleteDraftDialog
* action after clicking 'Share' icon
* tooltips for icons and button

## TODO

- backend action after clicking delete draft

Resolves  #3544

### CODE REVIEW

File: src/modules/dashboard/components/Dialogs/DeleteDraftDialog/DeleteDraftDialog.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  display: block;
+  padding: 3px 0 20px;
+  font-size: var(--size-normal);
+  font-weight: var(--weight-normal);
+  line-height: 18px;
+  color: var(--temp-grey-blue-7);
+}
+
+.button {
+  composes: themeDanger from '~core/Button/Button.css';
```
Comment (danbr) [inline src/modules/dashboard/components/Dialogs/DeleteDraftDialog/DeleteDraftDialog.css:20]: Figma uses pink color for button (very similar to danger)

**Review by danbr — APPROVED** [review]
Looking good.
I left a couple of minor comments, but approving, well done.

**Review by arrenv — CHANGES_REQUESTED** [review]
Thank you @agnieszkajarosikloj, only a couple of things.


Can we improve the alignment of the icon and hover state.

download (31)

This is the design:
Screenshot 2022-07-01 105313

Also, please update the copy to the design:

Screenshot 2022-07-01 105541

**Review by arrenv — APPROVED** [review]
Thank you @agnieszkajarosikloj, looking good to me!

**Pull request merged**